types/dat.gui/index.d.ts

```diff
@@ -1,6 +1,6 @@
-// Type definitions for dat.GUI 0.7.3
+// Type definitions for dat.GUI 0.7.4
```

Comment:
You should not have to have a patch version here at all. You can remove `"dt-header": false` in `tslint.json` and change this to 0.7.